utils/release/npm-publish-package.mjs

```diff
@@ -68,7 +68,7 @@ await (async () => {
   let currentNpmVersion = new SemVer(
     privatePackage
       ? '0.0.0' // private package does not have a npm version, so we default to the 'lowest' possible
-      : await describeNpmTag(packageJson.name, 'latest')
```

Comment:
Our old process waited for QA approval before updating the `latest` tag, so I decided to keep that behaviour by making the release process here publish with the `beta` tag instead.

Pre-release versions are published with the `alpha` tag.